stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/ChatApi.kt

```diff
@@ -386,5 +387,8 @@ internal interface ChatApi {
     @CheckResult
     fun getSyncHistory(channelIds: List<String>, lastSyncAt: Date): Call<List<ChatEvent>>
 
+    @CheckResult
+    fun getSyncHistory(channelIds: List<String>, lastSyncAt: ExactDate): Call<List<ChatEvent>>
```

Comment:
`ChatApi` is an internall interface, we don't need to support an extra method here. 
We can change method signature to get an String instead with the `lastSyncAt`.
If we do this change, we don't really need `ExactDate` neither `ExactDataUpstream` and its adapters

Reply:
We still need `ExactDate` for downstream